scripts/flattener.sh

```diff
@@ -1,6 +1,7 @@
-npx truffle-flattener contracts/DXswapFactory.sol > contracts/.flattened/DXswapFactory.sol
-npx truffle-flattener contracts/DXswapPair.sol > contracts/.flattened/DXswapPair.sol
-npx truffle-flattener contracts/DXswapERC20.sol > contracts/.flattened/DXswapERC20.sol
-npx truffle-flattener contracts/DXswapDeployer.sol > contracts/.flattened/DXswapDeployer.sol
-npx truffle-flattener contracts/DXswapFeeSetter.sol > contracts/.flattened/DXswapFeeSetter.sol
-npx truffle-flattener contracts/DXswapFeeReceiver.sol > contracts/.flattened/DXswapFeeReceiver.sol
+mkdir -p contracts/.flattened
```

Comment:
Not sure if this works on non-UNIX systems, maybe let's replace it with a Hardhat script/task? Maybe we can actually remove and use `hardhat flatten SOURCE_FILE` when needed.

Reply:
Removed with [115edb4](https://github.com/levelkdev/dxswap-core/pull/45/commits/115edb4b22578fef425b1c5463c8ae6016087403)